<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TouchScreen</class>
 <widget class="QDialog" name="TouchScreen">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>480</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>800</width>
     <height>0</height>
    </size>
   </property>
   <property name="palette">
    <palette>
     <active>
      <colorrole role="Mid">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>194</red>
         <green>198</green>
         <blue>189</blue>
        </color>
       </brush>
      </colorrole>
     </active>
     <inactive>
      <colorrole role="Mid">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>194</red>
         <green>198</green>
         <blue>189</blue>
        </color>
       </brush>
      </colorrole>
     </inactive>
     <disabled>
      <colorrole role="Mid">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>194</red>
         <green>198</green>
         <blue>189</blue>
        </color>
       </brush>
      </colorrole>
     </disabled>
    </palette>
   </property>
   <property name="font">
    <font>
     <family>Calibri</family>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QTabBar::tab { height: 70px; width: 400px; }</string>
   </property>
   <property name="tabShape">
    <enum>QTabWidget::Rounded</enum>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <property name="iconSize">
    <size>
     <width>40</width>
     <height>23</height>
    </size>
   </property>
   <property name="elideMode">
    <enum>Qt::ElideNone</enum>
   </property>
   <property name="tabBarAutoHide">
    <bool>false</bool>
   </property>
   <widget class="QWidget" name="tab">
    <property name="font">
     <font>
      <family>Calibri</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <attribute name="title">
     <string>MainMenu</string>
    </attribute>
    <widget class="QPushButton" name="TurnOnBtn">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>371</width>
       <height>321</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">#TurnOnBtn {
background-color: transparent;
border-image: url(:MyFiles/PowerOff.gif);
background: none;
border: none;
background-repeat: none;
}
</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="EDFAOff">
     <property name="geometry">
      <rect>
       <x>570</x>
       <y>150</y>
       <width>71</width>
       <height>61</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">#EDFAOff {
background-color: transparent;
border-image: url(:MyFiles/Red.png);
background: none;
border: none;
background-repeat: none;
}
</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>150</y>
       <width>141</width>
       <height>81</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Calibri</family>
       <pointsize>16</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Amplifier:</string>
     </property>
    </widget>
    <widget class="QPushButton" name="LaserQuestOff">
     <property name="geometry">
      <rect>
       <x>570</x>
       <y>50</y>
       <width>71</width>
       <height>61</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">#LaserQuestOff {
background-color: transparent;
border-image: url(:MyFiles/Red.png);
background: none;
border: none;
background-repeat: none;
}
</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>40</y>
       <width>141</width>
       <height>81</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Calibri</family>
       <pointsize>16</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Laser:</string>
     </property>
    </widget>
    <widget class="QPlainTextEdit" name="pteStatus">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>340</y>
       <width>371</width>
       <height>41</height>
      </rect>
     </property>
     <property name="mouseTracking">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>260</y>
       <width>131</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Calibri</family>
       <pointsize>16</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Stable:</string>
     </property>
    </widget>
    <widget class="QPushButton" name="StableOn">
     <property name="geometry">
      <rect>
       <x>570</x>
       <y>250</y>
       <width>71</width>
       <height>61</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">#StableOn {
background-color: transparent;
border-image: url(:MyFiles/Green.png);
background: none;
border: none;
background-repeat: none;
}
</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="LaserQuestOn">
     <property name="geometry">
      <rect>
       <x>570</x>
       <y>50</y>
       <width>71</width>
       <height>61</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">#LaserQuestOn {
background-color: transparent;
border-image: url(:MyFiles/Green.png);
background: none;
border: none;
background-repeat: none;
}
</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="EDFAOn">
     <property name="geometry">
      <rect>
       <x>570</x>
       <y>150</y>
       <width>71</width>
       <height>61</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">#EDFAOn {
background-color: transparent;
border-image: url(:MyFiles/Green.png);
background: none;
border: none;
background-repeat: none;
}
</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="StableOff">
     <property name="geometry">
      <rect>
       <x>570</x>
       <y>250</y>
       <width>71</width>
       <height>61</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">#StableOff {
background-color: transparent;
border-image: url(:MyFiles/Red.png);
background: none;
border: none;
background-repeat: none;
}
</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="TurnOffBtn">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>371</width>
       <height>321</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">#TurnOffBtn {
background-color: transparent;
border-image: url(:MyFiles/PowerOn.gif);
background: none;
border: none;
background-repeat: none;
}
</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="LaserPortOff">
     <property name="geometry">
      <rect>
       <x>670</x>
       <y>50</y>
       <width>71</width>
       <height>61</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">#LaserPortOff{
background-color: transparent;
border-image: url(:MyFiles/Red.png);
background: none;
border: none;
background-repeat: none;
}
</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="EDFAPortOff">
     <property name="geometry">
      <rect>
       <x>670</x>
       <y>150</y>
       <width>71</width>
       <height>61</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">#EDFAPortOff {
background-color: transparent;
border-image: url(:MyFiles/Red.png);
background: none;
border: none;
background-repeat: none;
}
</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="LaserPortOn">
     <property name="geometry">
      <rect>
       <x>670</x>
       <y>50</y>
       <width>71</width>
       <height>61</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">#LaserPortOn{
background-color: transparent;
border-image: url(:MyFiles/Green.png);
background: none;
border: none;
background-repeat: none;
}
</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="EDFAPortOn">
     <property name="geometry">
      <rect>
       <x>670</x>
       <y>150</y>
       <width>71</width>
       <height>61</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">#EDFAPortOn{
background-color: transparent;
border-image: url(:MyFiles/Green.png);
background: none;
border: none;
background-repeat: none;
}
</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>570</x>
       <y>20</y>
       <width>221</width>
       <height>33</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="label_6">
        <property name="font">
         <font>
          <family>Calibri</family>
          <pointsize>10</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Emission:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_8">
        <property name="font">
         <font>
          <family>Calibri</family>
          <pointsize>10</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Port:</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <zorder>layoutWidget</zorder>
    <zorder>EDFAPortOn</zorder>
    <zorder>EDFAPortOff</zorder>
    <zorder>LaserPortOn</zorder>
    <zorder>TurnOffBtn</zorder>
    <zorder>EDFAOn</zorder>
    <zorder>LaserQuestOn</zorder>
    <zorder>TurnOnBtn</zorder>
    <zorder>EDFAOff</zorder>
    <zorder>label_2</zorder>
    <zorder>LaserQuestOff</zorder>
    <zorder>label</zorder>
    <zorder>pteStatus</zorder>
    <zorder>label_7</zorder>
    <zorder>StableOn</zorder>
    <zorder>StableOff</zorder>
    <zorder>LaserPortOff</zorder>
   </widget>
   <widget class="QWidget" name="tab_2">
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <attribute name="title">
     <string>Settings:</string>
    </attribute>
    <widget class="QPushButton" name="Save">
     <property name="geometry">
      <rect>
       <x>600</x>
       <y>310</y>
       <width>131</width>
       <height>81</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Comic Sans MS</family>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">#Save {
background-color: transparent;
border-image: url(:MyFiles/Save.gif);
background: none;
border: none;
background-repeat: none;
}
</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>30</y>
       <width>731</width>
       <height>252</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="label_3">
          <property name="font">
           <font>
            <family>Calibri</family>
            <pointsize>16</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>EDFA Power(dBm):</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDoubleSpinBox" name="powerDb">
          <property name="font">
           <font>
            <family>Calibri</family>
            <pointsize>36</pointsize>
           </font>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
          <property name="buttonSymbols">
           <enum>QAbstractSpinBox::NoButtons</enum>
          </property>
          <property name="showGroupSeparator" stdset="0">
           <bool>false</bool>
          </property>
          <property name="decimals">
           <number>2</number>
          </property>
          <property name="maximum">
           <double>999.990000000000009</double>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDoubleSpinBox" name="PowerBox">
          <property name="font">
           <font>
            <family>Calibri</family>
            <pointsize>36</pointsize>
           </font>
          </property>
          <property name="buttonSymbols">
           <enum>QAbstractSpinBox::PlusMinus</enum>
          </property>
          <property name="decimals">
           <number>2</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label_5">
          <property name="font">
           <font>
            <family>Calibri</family>
            <pointsize>16</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Wavelength(nm):</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDoubleSpinBox" name="wavelengthDb">
          <property name="font">
           <font>
            <family>Calibri</family>
            <pointsize>36</pointsize>
           </font>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
          <property name="buttonSymbols">
           <enum>QAbstractSpinBox::NoButtons</enum>
          </property>
          <property name="showGroupSeparator" stdset="0">
           <bool>false</bool>
          </property>
          <property name="decimals">
           <number>3</number>
          </property>
          <property name="maximum">
           <double>999.999000000000024</double>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDoubleSpinBox" name="wavelengthSb">
          <property name="font">
           <font>
            <family>Calibri</family>
            <pointsize>36</pointsize>
           </font>
          </property>
          <property name="buttonSymbols">
           <enum>QAbstractSpinBox::PlusMinus</enum>
          </property>
          <property name="decimals">
           <number>3</number>
          </property>
          <property name="maximum">
           <double>999.999000000000024</double>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
    <widget class="QPushButton" name="Load">
     <property name="geometry">
      <rect>
       <x>440</x>
       <y>327</y>
       <width>121</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>Reset To Saved</string>
     </property>
    </widget>
    <zorder>layoutWidget</zorder>
    <zorder>Save</zorder>
    <zorder>Load</zorder>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
